core/services/fluxmonitor/flux_monitor.go

```diff
@@ -355,17 +371,21 @@ type PollingDeviationChecker struct {
 	store          *store.Store
 	fluxAggregator contracts.FluxAggregator
 	runManager     RunManager
+	logBroadcaster eth.LogBroadcaster
 	fetcher        Fetcher
+	flagsContract  *contracts.Flags
```

Comment:
Ideally this would be a pointer because we should only need one Flags contract in total, shared across all PDCs. But, alas... I couldn't get it setup that way because of some issue with the eth client mock.